tests/network/vmi_multus.go

```diff
@@ -576,6 +580,45 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 			})
 		})
 
+		Context("kubectl get vmi test for primary and secondary interfaces", func() {
+			It("should verify vmi ip value match primary interface ip value", func() {
+				Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault, "linux-br", bridge10CNIType, bridge10Name, 0, "", bridge10MacSpoofCheck)).To(Succeed())
+
+				vmi := libvmi.NewCirros(libvmi.WithNetwork(v1.DefaultPodNetwork()))
+				vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*newVMISpecIfaceWithBridgeBinding("nic1"), *v1.DefaultBridgeNetworkInterface(), *newVMISpecIfaceWithBridgeBinding("nic2")}
+				vmi.Spec.Networks = append(vmi.Spec.Networks, *newVMISpecMultusNetwork("nic1", "linux-br"), *newVMISpecMultusNetwork("nic2", "linux-br"))
+
+				vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				vmi = tests.WaitForSuccessfulVMIStart(vmi)
+
+				var primaryIfaceIP string
+				for _, currStatusIface := range vmi.Status.Interfaces {
+					if currStatusIface.Name == "default" {
+						primaryIfaceIP = currStatusIface.IP
+					}
+				}
```

Comment:
Please consider a helper function (that can be defined under `libnet`):
`primaryIfaceStatus := lookupIfaceStatusByName(defaultPodNetworkName)`

If it is not found, it should fail.

Reply:
added helper file libnet/interface.go